runtimes/web/src/ui/menu-overlay.ts

```diff
@@ -243,11 +258,16 @@ export class MenuOverlay extends LitElement {
     }
 
     render () {
+        if (!this.vibrateLevel) {
+            this.vibrateLevel = this.app.vibrateLevel[0];
```

Comment:
I tried setting this in the constructor, but looks like `app` is not yet ready, so moved it to here.